megamek/src/megamek/server/GameManager.java

```diff
@@ -4930,6 +4936,7 @@ private Vector<Report> processCrash(Entity entity, int vel, Coords c) {
             crateredElevation = h.getLevel() - 2;
         }
         if (entity instanceof Dropship) {
+            Dropship ds = (Dropship) entity;
```

Comment:
Why are we declaring this variable and not using it?

Reply:
Oops, good eye.  I think I was looking at replacing some casts, but it may no longer be necessary.